frontend/src/requests/FinnHub-Requests.js

```diff
@@ -0,0 +1,49 @@
+
+export function stockPricesRequest(symbol, range, start, end, setFunction) {
+    const finnhub = require('finnhub');
+    const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+    api_key.apiKey = process.env.FINNHUB_API_KEY;
+    const finnhubClient = new finnhub.DefaultApi()
+    finnhubClient.stockCandles(symbol, range, start, end, async (error, data, response) => {
```

Comment:
Remove async, I mistakenly left it there. I don't think it makes any difference.